### ISSUE TITLE
RF output connector

### ISSUE THREAD

**Issue by g4ixt** [bg3mdo/budget_23cm_band_RF_power_amplifier#1]
Have you tried using an N-type connector on the output?  That's a lot of power at 1.3GHz to be going through an SMA connector (although within the rating for good quality ones).  For your double transistor board (that I saw on Twitter you are working on), it will be very near the maximum power handling of the SMA connector.

For your hybrid combiner (on Twitter) you could perhaps fit a diode detector at the ballast load port, so that if there is a power imbalance or phase error you can give an indication or logic "trip" to save blowing it up.

I am going to try building one of these and have ordered some transistors via ali.

73 de G4IXT

**Comment by bg3mdo** [comment]
Dear Ian,

Thanks for your email!

For the output, we did try the N-type connector. N-type connector is much
better than SMA for large power handling. For the double transistor one,
the output SMA is for testing purposes - tuning each PA. After done the
tuning, the SAM connector will be removed from the board, and feeding the
output directly into an FR4 substrate coupler.

We have fit two directional couplers at the output side of the branch line
coupler to monitor the imbalance. We have made the following 30dB cable
coupler - milling two cables and solder together:

[image: dcoupler.png]

The transistor is cheaper enough, hope you have fun with the building
process!

VY 73.

Cheers,
BG3MDO Yuan
